ci/docker/Dockerfile.build.android_arm64

```diff
@@ -18,9 +18,16 @@
 #
 # Dockerfile to build MXNet for Android ARM64/ARMv8
 
+FROM ccache/build.ubuntu as ccachebuilder
+
+COPY install/ubuntu_install_ccache.sh /work/
+RUN /work/ubuntu_install_ccache.sh
+
 FROM dockcross/base:latest
 MAINTAINER Pedro Larroy "pllarroy@amazon.com"
 
+COPY --from=ccachebuilder /usr/local/bin/ccache /usr/local/bin/ccache
```

Comment:
what does this do? can we add a comment?

Reply:
Will do.